zcash_transparent/src/address.rs

```diff
@@ -14,7 +14,9 @@ use zcash_encoding::Vector;
 /// Most of the opcodes are unused by this crate, but we define them so that the alternate
 /// `Debug` impl for [`Script`] renders correctly for unexpected scripts.
 #[derive(Debug)]
-enum OpCode {
+
+// Maya Protocol integration needs OpCode to be public for add_op_return_output implementation
+pub enum OpCode {
```

Comment:
```suggestion
pub(crate) enum OpCode {
```

I think the opcodes are likely to end up fully `pub` eventually, but probably not until the switch to the Rust implementation of zcash_script.

Reply:
this also lacks a CHANGELOG entry if they went `pub`